app/assets/javascripts/discourse/app/templates/discovery/list.hbs

```diff
@@ -0,0 +1,50 @@
+<Discovery::Layout
+  @category={{this.model.category}}
+  @createTopicDisabled={{this.createTopicDisabled}}
+>
+  <:navigation>
+    <Discovery::Navigation
+      @category={{this.model.category}}
+      @tag={{this.model.tag}}
+      @additionalTags={{this.model.additionalTags}}
+      @filterType={{this.model.filterType}}
+      @noSubcategories={{this.model.noSubcategories}}
+      @canBulkSelect={{this.canBulkSelect}}
+      @bulkSelectHelper={{this.bulkSelectHelper}}
+      @createTopic={{this.createTopic}}
+      @createTopicDisabled={{this.createTopicDisabled}}
+      @canCreateTopicOnTag={{this.model.canCreateTopicOnTag}}
+      @toggleTagInfo={{this.toggleTagInfo}}
+      @tagNotification={{this.model.tagNotification}}
+    />
+  </:navigation>
+
+  <:header>
+    {{#if this.model.subcategoryList}}
+      <Discovery::CategoriesDisplay
+        @categories={{this.model.subcategoryList.categories}}
+        @parentCategory={{this.model.subcategoryList.parentCategory}}
+      />
+    {{/if}}
+    {{#if this.showTagInfo}}
+      <TagInfo @tag={{this.model.tag}} @list={{this.model.list}} />
+    {{/if}}
+  </:header>
+
+  <:list>
+    {{this.setTrackingTopicList this.model.list}}
```

Comment:
This feels a bit… foreign 😅 But the whole topic-list-tracker code should probably be in a service, for starters. We can come back to that later.

Reply:
Yeah agreed, it is weird. Basically, I want an observer on the `model.list` property 😅 

Will see if I can find a cleaner way to do this 👀

Reply:
Moved to the routes. Some duplication, but less weird.